pkgs/desktops/cursor/fuchsia-pop/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl }:
+
+
+stdenv.mkDerivation rec {
+  pname = "fuchsia-pop";
+  version = "2.0.0";
+
+  src =  fetchurl {
```

Comment:
Looks like the repo has instructions to build from source. Maybe try that instead?